pages/common/twurl.md

```diff
@@ -0,0 +1,28 @@
+# twurl
+
+> Curl like command but tailored specifically for the Twitter API.
+> More information: <https://github.com/twitter/twurl>.
+
+- Authorize twurl to access a twitter account:
+
+`twurl authorize --consumer-key {{twitter_api_key}} --consumer-secret {{twitter_api_secret}}`
+
+- Make a GET request to an API endpoint:
+
+`twurl -X GET {{twitter_api_endpoint}}`
+
+- Make a POST request to an API endpoint:
+
+`twurl -X POST -d '{{endpoint_params}}' {{twitter_api_endpoint}}`
+
+- Upload a media:
```

Comment:
Where does it upload to specifically. Does it post the media as a new tweet? (upload to Twitter is a little unspecific)

Reply:
Agreed, also _a media_ doesn't sound good.
```suggestion
- Upload media to Twitter:
```